definitions/npm/pad_v1.x.x/flow_>=v0.27.x/pad_v1.x.x.js

```diff
@@ -0,0 +1,13 @@
+type padOptions = {
```

Comment:
Everything which would end up in the global scope has to be namespaced...
In this case it would be `$npm$pad$padOptions` or maybe just `$npm$pad$Options` :-)